packages/frontend/src/components/Explorer/ResultsCard/ColumnHeaderContextMenu.tsx

```diff
@@ -41,6 +41,25 @@ const ContextMenu: FC<ContextMenuProps> = ({
         (context) => context.actions.removeActiveField,
     );
 
+    const additionalMetrics = useExplorerContext(
+        (context) =>
+            context.state.unsavedChartVersion.metricQuery.additionalMetrics,
+    );
+
+    const additionalMetric = useMemo(
+        () =>
+            !!additionalMetrics &&
+            !!item &&
+            additionalMetrics.find((am) => am.name === item.name),
```

Comment:
We should check field IDs since names aren't unique across tables.
```suggestion
            additionalMetrics.find((am) => getItemId(am) === getItemId(item)),
```

Reply:
🤦 thanks for spotting this!